modules/core/client/views/modal.client.view.html

```diff
@@ -0,0 +1,18 @@
+<div class="modal-header">
+  <h3 class="modal-title">{{modal.title}}</h3>
+</div>
+<div class="modal-body">
+  {{modal.body}}
+  <div data-ng-if="modal.items">
```

Comment:
We have another PR for data- handling and I think at the moment we can leave the `data-*` stuff in there as is until we get a better decision on that change.